core/dbt/deprecations.py

```diff
@@ -43,6 +43,13 @@ def show(self, *args, **kwargs) -> None:
             active_deprecations.add(self.name)
 
 
+class DispatchPackagesDeprecation(DBTDeprecation):
+    _name = 'dispatch-packages'
+    _description = '''\
+    to fill in: "{macro_name}"
```

Comment:
Doesn't have to be this exactly:
```suggestion
The 'packages' argument of adapter.dispatch() has been deprecated. Use the 'macro_namespace' argument instead.
  Raised during dispatch for: {macro_name}
  Read details: https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch
```
That docs link won't work right until we merge some docs PRs + move `next` into current site, but ¯\_(ツ)_/¯